stacks/apps/augury.yml

```diff
@@ -41,7 +41,6 @@ Parameters:
   SharedAuroraPostgresqlEndpoint: { Type: String }
 
 Conditions:
-  IsStaging: !Equals [!Ref EnvironmentType, Staging]
   IsProduction: !Equals [!Ref EnvironmentType, Production]
   IsPrimaryRegion: !Equals [!Ref RegionMode, Primary]
   HasRdsEndpoint: !Not [!Equals [!Ref SharedRdsPostgresqlEndpoint, ""]]
```

Comment:
@farski should i make any attempt to clean up these guards?  I see a mix of `Condition HasRdsEndpoint not used` and `Condition HasAuroraEndpoint not used` when I lint these new app stacks.

Reply:
I wouldn't worry about `HasRdsEndpoint` much, that will be going away once everything is off RDS in staging and prod.

`HasAuroraEndpoint` would be worth double checking at some point. I think anything that's using either Aurora endpoint value should be behind a `HasAuroraEndpoint` condition.